dev/MRTCore/azure-pipelines.yml

```diff
@@ -376,6 +376,14 @@ jobs:
       Contents: |
         mrm.h
       TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\include'
+
+  - task: CopyFiles@2
+    displayName: 'copy winrt generated files'
+    inputs:
+      SourceFolder: '$(MRTSourcesDirectory)\mrt\Microsoft.ApplicationModel.Resources\src\Generated Files'
+      Contents: |
+        *
+      TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\include'
```

Comment:
why?

Reply:
Needed for C++/WinRT in Win32 apps.

Reply:
doesn't feel right. winmd should be the only thing needed

Reply:
"To successfully reference a C++ Windows Runtime component from a C++ app, you need to use C++/WinRT to generate projection header files of your component. You can then include these header files in your app code to call your component." https://blogs.windows.com/windowsdeveloper/2019/04/30/enhancing-non-packaged-desktop-apps-using-windows-runtime-components/

Sample app: https://github.com/microsoft/RegFree_WinRT/tree/master/Cpp

Reply:
that means whoever consumes the component (i.e. app developer) needs to generate the headers, not we need to provide them in nuget.

Reply:
we no longer add the generated files to the nuget package. they are generated from the winmd on app build in the app folder.